### ISSUE TITLE
Problems with Platypus package with the Binh and Korn function

### ISSUE THREAD

**Issue by tuli01** [Project-Platypus/Platypus#26]
I am trying to learn the Platypus package. I only slightly modified the routine shown as the site example below
def belegundu(vars):
    x = vars[0]
    y = vars[1]
    return [-2*x + y, 2*x + y], [-x + y - 1, x + y - 7]

problem = Problem(2, 2, 2)
problem.types[:] = [Real(0, 5), Real(0, 3)]
problem.constraints[:] = "<=0"
problem.function = belegundu

algorithm = NSGAII(problem)
algorithm.run(10000)

to use the **Binh and Korn function** given here: https://en.wikipedia.org/wiki/Test_functions_for_optimization#Test_functions_for_multi-objective_optimization as shown below

from platypus import NSGAII, Problem, Real

def telegundu(vars):
    x = vars[2.5]
    y = vars[1.5]
    return [4*x**2 + 4*y**2,(x-5)**2+(y-5)**2], (x-5)**2+y**2-25, (x-8)**2+(y+3)**2-7.7]

problem = Problem(2, 2, 2)
problem.types[:] = [Real(0, 5), Real(0, 3)]
problem.constraints[:] =[ "<=0",">=0"]
problem.function = telegundu

algorithm = NSGAII(problem)
algorithm.run(10000)
for solution in algorithm.result:
    print(solution.objectives)

For some reason I get this error:
TypeError: unhashable type: 'list'
Any ideas?
Thank you

**Comment by dhadka** [comment]
Hi,

It seems it doesn't like using a list to input the constraints.  I'll need to investigate more how to fix this, but for now the following should work:

```python

    problem.constraints[0] = "<=0"
    problem.constraints[1] = ">=0"
```

I also changed these two lines to get the example working:

```python

    x = vars[0]
    y = vars[1]
```

Full version:

```python

from platypus import NSGAII, Problem, Real

def telegundu(vars):
    x = vars[0]
    y = vars[1]
    return [4*x**2 + 4*y**2,(x-5)**2+(y-5)**2], [(x-5)**2+y**2-25, (x-8)**2+(y+3)**2-7.7]

problem = Problem(2, 2, 2)
problem.types[:] = [Real(0, 5), Real(0, 3)]
problem.constraints[0] = "<=0"
problem.constraints[1] = ">=0"
problem.function = telegundu

algorithm = NSGAII(problem)
algorithm.run(10000)

for solution in algorithm.result:
    print(solution.objectives)
```

**Comment by tuli01** [comment]
Thank you

**Comment by dhadka** [comment]
Fixed in 04e6283c47778889da9e3047820931420b1ee7e2.  You can now pass in constraints as an array: `problem.constraints[:] = ["<=0", ">=0"]`.
